ConsentViewController/Classes/ConsentViewController.swift

```diff
@@ -490,6 +490,19 @@ import Reachability
         didMessageScriptLoad = true
         consentDelegate.onMessageReady(controller: self)
     }
+    
+    /// It will clear all the stored userDefaults Data
+    public func clearAllConsnetData() {
```

Comment:
There's a typo on the name, `cosnet` instead of `consent`

Reply:
Chnaged typo from `cosent` to `consent`